docs/faq.md

```diff
@@ -27,7 +27,8 @@ If a question you have is not answered below, please [submit an issue](/../../is
 [What would a Dockerfile for a Jib-built image look like?](#what-would-a-dockerfile-for-a-jib-built-image-look-like)\
 [How can I inspect the image Jib built?](#how-can-i-inspect-the-image-jib-built)\
 [I would like to run my application with a javaagent.](#i-would-like-to-run-my-application-with-a-javaagent)\
-[How can I tag my image with a timestamp?](#how-can-i-tag-my-image-with-a-timestamp)
+[How can I tag my image with a timestamp?](#how-can-i-tag-my-image-with-a-timestamp)\
+[How do I specify an image from a manifest list?](#how-do-i-specify-an-image-from-a-manifest-list)
```

Comment:
For the title, how about "a manifest list or an OCI index"? (But we can omit OCI index in the text for brevity.)

And how about "specify OS and architecture of a base image from..."?

Reply:
I'm not really sure about the title.

`How do I specify a platform in the manifest list (or OCI index) of a base image?"

Reply:
Yeah, I think "platform" is a good keyword for search.